docusaurus/src/theme/DocSidebar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import OriginalDocSidebar from '@theme-original/DocSidebar';
+
+const WEB_LINKS = [
+  ['Contact Support', 'https://getstream.io/contact/support/'],
+  ['Maker Account', 'https://getstream.io/maker-account/'],
+  ['Mobile Chat Kit', 'https://getstream.io/chat/ux-kit/']
+]
+
+export default function DocSidebar(props) {
+  return <>
+  <OriginalDocSidebar {...props} sidebar={props.sidebar.map(category => ({ ...category, collapsed: false }))}/>
```

Comment:
can you memoize the sidebar map logics? like: 
```
const sidebar = useMemo(() => props.sidebar.map(category => ({ ...category, collapsed: false })) , props.sidebar)
```

and then `sidebar={sidebar}`.

Just to make sure it wont run when its not needed

Reply:
Good catch!